src/Halogen/Data/Slot.purs

```diff
@@ -112,13 +117,14 @@ slots
   => Proxy sym
   -> SlotStorage slots slot
   -> Map s (slot query output)
-slots sym (SlotStorage m) = Map.foldSubmap Nothing Nothing go m
```

Comment:
I don't think this needs to be removed -- `foldSubmap` still exists in ordered-collections

Reply:
yes, I have tried to impl it with SemigroupMap but it didnt work

but it required Semigroup on v

```
instance semigroupSemigroupMap :: (Ord k, Semigroup v) => Semigroup (SemigroupMap k v) where
  append (SemigroupMap l) (SemigroupMap r) = SemigroupMap (unionWith append l r)
```

Reply:
you can use the Alternate newtype to get back the old behavior: https://pursuit.purescript.org/packages/purescript-control/4.2.0/docs/Data.Monoid.Alternate#t:Alternate